tests/functional/raw-node/test/GCP/object/delete.js

```diff
@@ -0,0 +1,116 @@
+const assert = require('assert');
+const async = require('async');
+const { GCP } = require('../../../../../../lib/data/external/GCP');
+const { makeGcpRequest } = require('../../../utils/makeRequest');
+const { gcpRequestRetry } = require('../../../utils/gcpUtils');
+const { getRealAwsConfig } =
+    require('../../../../aws-node-sdk/test/support/awsConfig');
+
+const credentialOne = 'gcpbackend';
+
+describe('GCP: DELETE Object', function testSuite() {
+    this.timeout(30000);
+    const config = getRealAwsConfig(credentialOne);
+    const gcpClient = new GCP(config);
+
```

Comment:
Seems like you could have a `beforeEach` hook here that creates the bucket rather than do it twice in the nested `describe` statements.

Reply:
Thanks! will restructure the hooks.